cmd/create/admin/cmd.go

```diff
@@ -186,6 +182,8 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	reporter.Infof("Admin account has been added to cluster '%s'.", clusterKey)
+	reporter.Infof("An Identity Provider named 'Cluster-Admins' from type HTPasswdIdentityProvider, " +
```

Comment:
I would change it to I: An HTPasswd Identity Provider was created with the name 'Cluster-Admin' cc: @arendej